tests/test_yaml.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import pytest
+
+import helper
+from rpmlb.yaml import Yaml
+
+
+@pytest.fixture
+def valid_yaml():
+    """A Yaml object with a valid YAML file."""
+    return Yaml(helper.get_valid_recipe_file())
+
+
+def test_init_loads_file():
+    yaml = Yaml(helper.get_valid_recipe_file())
```

Comment:
An error of a fixture is error of it's test. Please change it so it uses the fixture.